pkg/kopia/connect.go

```diff
@@ -75,19 +74,9 @@ func ConnectToAPIServer(
 		},
 	}
 
-	err = poll.WaitWithBackoff(ctx, backoff.Backoff{
-		Factor: 2,
-		Jitter: false,
-		Min:    100 * time.Millisecond,
-		Max:    180 * time.Second,
-	}, func(c context.Context) (bool, error) {
-		if err := repo.ConnectAPIServer(ctx, defaultConfigFilePath, serverInfo, string(passphrase), opts); err != nil {
-			return false, nil
-		}
-		return true, nil
-	})
-	if err != nil {
-		return errors.Wrap(err, "Failed to backup data to Kopia API server")
+	// TODO(@pavan): Modify this to use custom config file path, if required
+	if err = repo.ConnectAPIServer(ctx, defaultConfigFilePath, serverInfo, string(passphrase), opts); err != nil {
+		return errors.Wrap(err, "Failed connecting to the Kopia API server")
```

Comment:
@pavannd1 why don't we want to retry? Probably I am missing a context here

Reply:
Fixed this.